sources/packages/api/src/database/entities/student-assessment.model.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import {
+  Application,
+  AssessmentStatus,
+  EducationProgramOffering,
+  StudentAppeal,
+} from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { AssessmentTriggerType } from "./assessment-trigger.type";
+import { RecordDataModel } from "./record.model";
+import { StudentScholasticStanding } from "./student-scholastic-standing.model";
+
+/**
+ * Represents all the assessments and reassessments processed for a particular
+ * Student Application. When a assessment/reassessment is needed it will use
+ * the information present in the application itself combined with the additional
+ * data present in this table (e.g. student appeals and scholastic standing changes).
+ */
+@Entity({ name: TableNames.StudentAssessment })
```

Comment:
same here

Reply:
We do have both but I agree to keep the constants more aligned with the real table name.
![image](https://user-images.githubusercontent.com/61259237/157290052-be2669f4-ca2a-48af-b7f2-010e6b5a1519.png)

Reply:
Changed!